src/open_r1/rewards.py

```diff
@@ -76,8 +75,8 @@ def accuracy_reward(completions, solution, **kwargs):
                 print(f"verify failed: {e}, answer: {answer_parsed}, gold: {gold_parsed}")
                 reward = 0.0
         else:
-            # If the gold solution is not parseable, we reward 1 to skip this example
-            reward = 1.0
+            # If the gold solution is not parseable, we reward 0 to skip this example
+            reward = 0.0
```

Comment:
Maybe `None` if we want the example to be skipped?

Reply:
Ah yes, great idea!